variables.tf

```diff
@@ -23,6 +23,16 @@ DESCRIPTION
   nullable    = false
 }
 
+variable "subnets" {
```

Comment:
There is an existing "shared_subnet_id" variable that resources currently use, which likely need to be updated with the changes in this PR

Reply:
Have done, thanks!